requirements_gpu.txt

```diff
@@ -1,7 +1,8 @@
 # Common Requirements
 -r requirements_common.txt
 
-# Torch GPU Version
+# Torch CUDA 11-compatible GPU Version, from https://pytorch.org/get-started/locally/
 --find-links https://download.pytorch.org/whl/torch_stable.html
-torch==1.5.0
-torchvision==0.6.0
\ No newline at end of file
+torch==1.8.1+cu111
+torchvision==0.9.1+cu111
+torchaudio==0.8.1
```

Comment:
I'm not sure why it's there either tbh! I haven't tested all the combinations.

It might be one of those situations where not pinning torchaudio might accidentally bump torch to a different version if some other library depends on it. But maybe we can live with that?

Reply:
I'll take it out.